epub.go

```diff
@@ -557,6 +559,50 @@ func (e *Epub) Title() string {
 	return e.title
 }
 
+// EmbedImages download <img> tags in EPUB and modify body to show images
+// file inside of EPUB:
+// ../ImageFolderName/internalFilename
+//
+// The image source should either be a URL, a path to a local file, or an embedded data URL; in any
+// case, the image file will be retrieved and stored in the EPUB.
+//
+// The internal filename will be used when storing the image file in the EPUB
+// and must be unique among all image files. If the same filename is used more
+// than once, FilenameAlreadyUsedError will be returned. The internal filename is
+// optional; if no filename is provided, one will be generated.
+
+// Just call EmbedImages() after section added
```

Comment:
This looks great! one last question: do you think it's worth mentioning in the documentation for this method that it won't return an error if any of the images fail to download?